schematic/utils/df_utils.py

```diff
@@ -4,17 +4,58 @@
 
 import logging
 from copy import deepcopy
-from time import perf_counter
-from typing import Union, Any, Optional
 from datetime import datetime
+from time import perf_counter
+from typing import Any, Optional, Union
+
 import dateparser as dp
-import pandas as pd
 import numpy as np
+import pandas as pd
 from pandarallel import pandarallel  # type: ignore
 
+# pylint:disable=no-name-in-module
+from pandas._libs.parsers import STR_NA_VALUES  # type: ignore
+
+STR_NA_VALUES_FILTERED = deepcopy(STR_NA_VALUES)
+
+try:
```

Comment:
May I ask why this is necessary?

Reply:
When using `.remove` on a set it will raise a KeyError if not found. I wanted to add this in and future proof the code and the super slim chance that some version that a user has of pandas doesn't have "None" in the set.

I know the project toml has pandas 2.x.x set, but if you were to manually go back to 1.x.x this code would break without the try/except